server/api/impl/store/apps.go

```diff
@@ -4,97 +4,190 @@
 package store
 
 import (
+	"crypto/sha1" // nolint:gosec
+	"encoding/json"
+	"fmt"
+	"sync"
+
+	"github.com/pkg/errors"
+
 	"github.com/mattermost/mattermost-plugin-apps/apps"
 	"github.com/mattermost/mattermost-plugin-apps/server/api"
 	"github.com/mattermost/mattermost-plugin-apps/server/utils"
 )
 
-type AppStore struct {
+// appStore combines installed and builtin Apps.  The installed Apps are stored
```

Comment:
as an observation, it is maybe worth noticing that for apps - we store the builtin ones in memory and the non-builtin/external/global(?) in KV store while for manifests we store the global ones in memory and the local ones in the KV store.

Reply:
Apps are only "installed" or builtin (also pre-installed). The manifest store is possibly going to change somewhat after discussing with @iomodo and @hanzei, tempted to add another override layer, `dev-manifests.json` that'd be combined with `manifests.json` in memory. 1/5 we keep the HA-capable "local" manifest store for future use-cases.